client/packages/programs/src/api/hooks/useDeleteSelectedImmunisationPrograms.ts

```diff
@@ -40,7 +40,7 @@ export const useDeleteSelectedImmunisationPrograms = () => {
       selectedRows.map(
         async id => await mutateAsync({ immunisationProgramId: id })
       )
-    ).then(() => queryClient.invalidateQueries([PROGRAM]));
```

Comment:
I think I had deliberately shared keys to allow for the possiblity that an immunisation program could be a normal program at the same time. Makes sense to me that we might want to procure vaccines, track their usage, etc under the same program. I think changing it is fine for now, and probably for future to realistically, shouldn't be deleted often...